src/main/java/no/nav/veilarbvedtaksstotte/service/OyeblikksbildeService.java

```diff
@@ -67,4 +74,21 @@ void lagreOyeblikksbilde(String fnr, long vedtakId) {
 
         oyeblikksbilde.forEach(oyeblikksbildeRepository::upsertOyeblikksbilde);
     }
+
+    public String mapEgenvurderingTilGammelStruktur(EgenvurderingResponseDTO egenvurderingResponseDTO) {//public for test
```

Comment:
Frykter at hva "GammelStruktur" betyr er glemt om noen måneder. Kanskje legge til en kommentar som beskriver litt konteksten rundt?

Reply:
Alternativt kan man endre metoden til å hete f.eks. `mapTilEgenvurderingData`, for så å trekke `toJson` opp et hakk. Da blir det iallfall tydeligere for meg at vi tilpasser (mapper) dataen til et annet format (som vi "støtter").

Reply:
Enig i at GammelStruktur ikke gir mening etter en tid. Jeg synes det blir litt "rotete å trekke toJson opp fordi det vil måtte være en null sjekk osv så jeg liker at er contained inne i metoden så derfor lurer jeg på om du tror det går fint å bare rename metoden til: `mapToEgenvurderingDataJson` ? 🤷‍♀️ :D